src/webui/www/private/views/cookies.html

```diff
@@ -0,0 +1,183 @@
+<style>
+    #manageCookiesContainer {
+        margin: 10px;
+
+        .buttonContainer {
+            text-align: center;
+            margin-top: 1em;
+        }
+    }
+
+    #manageCookiesTable {
+        tbody {
+            td input {
+                width: 140px;
+            }
+
+            td.expDate input {
+                width: 190px;
+            }
+        }
+
+        thead td {
+            width: 150px;
+
+            .expDate {
+                width: 200px;
+            }
+
+            .actionColumn {
+                width: 30px;
+            }
+        }
+
+        .removeCookie,
+        .addCookie {
+            height: 16px;
+            width: 16px;
+            padding-left: 10px;
+            vertical-align: middle;
+            cursor: pointer;
+        }
+    }
+
+</style>
+
+<div id="manageCookiesContainer">
+    <table id="manageCookiesTable">
+        <thead>
+            <tr>
+                <td class="domain">QBT_TR(Domain)QBT_TR[CONTEXT=CookiesDialog]</td>
+                <td class="path">QBT_TR(Path)QBT_TR[CONTEXT=CookiesDialog]</td>
+                <td class="name">QBT_TR(Name)QBT_TR[CONTEXT=CookiesDialog]</td>
+                <td class="value">QBT_TR(Value)QBT_TR[CONTEXT=CookiesDialog]</td>
+                <td class="expDate">QBT_TR(Expiration Date)QBT_TR[CONTEXT=CookiesDialog]</td>
+                <td class="actionColumn"></td>
+            </tr>
+        </thead>
+        <tbody>
+            <tr class="invisible newRow">
+                <td class="domain"><input type="text" aria-label="QBT_TR(Domain)QBT_TR[CONTEXT=CookiesDialog]" autocomplete="off" autocorrect="off" autocapitalize="none"></td>
+                <td class="path"><input type="text" aria-label="QBT_TR(Path)QBT_TR[CONTEXT=CookiesDialog]" autocomplete="off" autocorrect="off" autocapitalize="none"></td>
+                <td class="name"><input type="text" aria-label="QBT_TR(Name)QBT_TR[CONTEXT=CookiesDialog]" autocomplete="off" autocorrect="off" autocapitalize="none"></td>
+                <td class="value"><input type="text" aria-label="QBT_TR(Value)QBT_TR[CONTEXT=CookiesDialog]" autocomplete="off" autocorrect="off" autocapitalize="none"></td>
+                <td class="expDate"><input type="datetime-local" aria-label="QBT_TR(Expiration Date)QBT_TR[CONTEXT=CookiesDialog]"></td>
+                <td><img class="removeCookie" src="images/list-remove.svg" onclick="qBittorrent.ManageCookies.deleteCookie(this);" alt="QBT_TR(Remove)QBT_TR[CONTEXT=CookiesDialog]"></td>
```

Comment:
In general, we should stop using `onclick` and use addEventListener from js side. It is cleaner IMO.

Reply:
AFAIK the `onclick` directive isn't deprecated, and there's no prior agreement in this repo to avoid `onclick`. There doesn't seem to be much difference between the two.

Reply:
>and there's no prior agreement in this repo to avoid onclick

I'm bringing the discussion now.

>There doesn't seem to be much difference between the two.

By using `addEventListener ` the separation of html and js is cleaner, js code won't be mixed within html anymore.

If you are interested, they have summarized here: https://stackoverflow.com/questions/6348494/addeventlistener-vs-onclick but the points made aren't too related to the code here. The only reason is my point above.

Reply:
I'm not sure it's worth it, there are much larger issues in the WebUI codebase - namely the use of mootools and MochaUI. I do think it's cleaner in this case so will make the change, but it's not something I think is worth spending  much time on.